applications/nrf_desktop/src/modules/ble_conn_params.c

```diff
@@ -23,11 +23,12 @@
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_CONN_PARAMS_LOG_LEVEL);
 
 #define CONN_INTERVAL_LLPM_US		1000   /* 1 ms */
-#define CONN_INTERVAL_LLPM_REG		0x0d01 /* 1 ms */
 #if (CONFIG_CAF_BLE_USE_LLPM && (CONFIG_BT_MAX_CONN >= 2))
  #define CONN_INTERVAL_BLE_REG		0x0008 /* 10 ms */
+ #define CONN_INTERVAL_BLE_US           10000
 #else
  #define CONN_INTERVAL_BLE_REG		0x0006 /* 7.5 ms */
+ #define CONN_INTERVAL_BLE_US           7500
```

Comment:
Can we use `BT_GAP_US_TO_CONN_INTERVAL` macro instead of defining `CONN_INTERVAL_BLE_REG` macro? That would simplify implementation and got rid of redundant definitions.